sales/hubspot.md

```diff
@@ -0,0 +1,57 @@
+---
+navTitle: HubSpot
+---
+
+## HubSpot
+
+We use [HubSpot](https://www.hubspot.com/) to track and manage all of our customer interactions.
+It enables the creation of customer contacts, and then logging of emails/notes associated to that customer.
+
+### Creating Contacts
+
+Given that we have multiple client-facing roles within the Company, it is important to document those that
+we talk to to ensure no crossover in sales and conversations.
+
+It is best practice to ensure a HubSpot contact is created prior to meeting with them, as soon as a
+call/conversation is scheduled.
+
+This minimises the chances of another FlowForge colleague arranging a
```

Comment:
Very fair feedback, and is my own misunderstanding of what we would be using HubSpot for. Although, this does still leave us with a hole of "people we've spoken to" not being stored in an easy to manage/search place.

Reply:
I am not clear on what is the issue? Why does Nick need to be aware of all conversations with people he knows.

Reply:
> 

I talk with a lot of people during my day. Networking is an important part of the job. Not sure why these conversations need to be recorded?

Reply:
More linked to, Nick has knowledge about that person, or may have spoken with that person about Node-RED in such detail that a call with them by another member of FlowForge is a waste of time. @knolleary for context and any further input that I'm misrepresenting from our chat the other day

Reply:
> Not sure why these conversations need to be recorded?

My thinking was to prevent two members of FlowForge reaching out to the same individual about the same topic/conversation. It was always embarrassing at IBM when a client told us that another member of IBM had been speaking to them recently, and we had no context about what was said. At a company of 300k+, that felt excusable, at a company of 10, I'm not sure it is.

Reply:
>  Once contacts are added into Hubspot, we will start marketing to them and I am not sure this will go over well with people we contacted for product input.

That isn't necessarily how we have been using it - rather using it as a more general CRM of people we are talking to, to avoid us cold-calling the same people in the NR community repeatedly. This would cover both sales opportunities as well as product research.

I have no issue changing things up to be more sales/marketing focussed in just as long as we know what we should be using when.

As Joe mentioned, I have seen a small number of occasions where we've reached out to people I know in the community where, given the opportunity, I could have provided some pre-call context on our side that may have helped make the call more productive. Or the call has come as a result of something *they* have shared on the forum and got our attention - so we need some amount of coordination as to who reaches out to them.

I have zero experience of using HubSpot or CRM tools prior to FF, so I don't have an instinct for what the right way to manage this is.

Reply:
I would not mix in interactions with the Node-RED community with HubSpot. As we scale up the sales and marketing, we will run the risk of having sales people go after Node-RED community members. As you can imagine this will not be good for FlowForge.

I would encourage us to think of Hubspot as being a marketing and sales pipeline where we are nuturing people through to becoming a customer. Once they are a customer we support them through HubSpot and develop a relationship to expand (ie. more sales).